lib/flow/fund.ts

```diff
@@ -34,7 +34,34 @@ transaction(address: Address, amount: UFix64) {
 }
 `
 
-type TokenType = "FLOW"
+const txFundAccountFUSD = `
+import FUSD from ${config.contractFUSD}
+import FungibleToken from ${config.contractFungibleToken}
+
+transaction(address: Address, amount: UFix64) {
+  let tokenMinter: &FUSD.MinterProxy
+  let tokenReceiver: &{FungibleToken.Receiver}
+
+  prepare(minterAccount: AuthAccount) {
+      self.tokenMinter = minterAccount
+          .borrow<&FUSD.MinterProxy>(from: FUSD.MinterProxyStoragePath)
+          ?? panic("No minter available")
+
+      self.tokenReceiver = getAccount(address)
+          .getCapability(/public/fusdReceiver)!
+          .borrow<&{FungibleToken.Receiver}>()
+          ?? panic("Unable to borrow receiver reference")
+  }
+
+  execute {
+      let mintedVault <- self.tokenMinter.mintTokens(amount: amount)
+
+      self.tokenReceiver.deposit(from: <-mintedVault)
```

Comment:
FungibleToken.Receiver expects a FungibleToken.Vault instead of a FUSD.Vault

```
error: mismatched types
  --> eeb46f4647bf4602f9091fae399712ac8e597351a7ebfebae82a530b1118de1d:23:39
   |
23 |       self.tokenReceiver.deposit(from: <-mintedVault)
   |                                        ^^^^^^^^^^^^^ expected `FungibleToken.Vault`, got `FUSD.Vault`
```